### DISCUSSION TITLE
Error running example in zapdos

### DISCUSSION THREAD

**Discussion by youareit125** [shannon-lab/zapdos#150] [Q&A Software Development]
Hi,

Tried to run Lymberopoulos_with_argon_metastables_2D_At1Torr.i using zapdos, after replacing few lines such as 'EfieldAdvectionElectrons' to 'EfieldAdvection', I now get an error saying, "The requested regular material property muem is declared as an AD property. Either retrieve it as an AD property with getADMaterialProperty or declare it as a regular property with declareProperty". I have tried to change   _muem(declareADProperty<Real>("muem")) line to _muem(declareProperty<Real>("muem")) in src/materials/GasElectronMoments.C but rebuliding fails. I tried to change "type = GasElectronMoments" to "type = AdGasElectronMoments" to use AdGasElectronMoments.C file but this seems like it doesn't do anything. Is there a way to fix this issue?

**Comment by csdechant** [comment]
Zapdos uses AD materials for all the mobility and diffusion coefficients, so changing the materials in the materials block from AD to regular will case a lot of problems. Depending on what lines you changes, you might be using a regular kernel, instead of an AD kernel. 

Also, `EfieldAdvectionElectrons` is an older kernel that Zapdos currently doesn't have (after the switch from to using AD materials, `EfieldAdvectionElectrons` became obsolete, compared to `EfieldAdvection`). When was the last time your local branch of Zapdos was updated? (Instructions for updating Zapdos can be found here: https://shannon-lab.github.io/zapdos/getting_started/installation.html)
